devel/gcli/Portfile

```diff
@@ -30,7 +30,11 @@ depends_lib-append    port:curl
 
 depends_build-append  port:bison \
                       port:flex \
-                      port:pkgconfig
+                      path:bin/pkg-config:pkgconfig
 
 use_autoreconf        yes
 
+# cc1: error: unrecognized command line option "-Wno-gnu-zero-variadic-macro-arguments"
+if {[string match *gcc-4.* ${configure.compiler}]} {
```

Comment:
Why not just blacklist gcc4?

Reply:
It builds with gcc-4.2.

AFAIK, the current policy is to support old compilers as long as that is trivially doable (and dropping a flag is easy enough).